File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Python/factorial.py
"""
/*Program 5: Write a C program to calculate the factorial of a given number.
Input: 5
Output:
The Factorial of 5 is: 120*/
"""

num = 5

fact = 1

for itr in range(2,num+1):
    fact = fact*itr
 
print("The Factorial of 5 is "+str(fact)) 

File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Java/Factorial.java
/*Program 5: Write a C program to calculate the factorial of a given number.
Input: 5
Output:
The Factorial of 5 is: 120*/

class Factorial{
	

	public static void main(String args[]){
		
		int num = 5;
		int fact = 1;
		for(int i = 2 ; i<=5 ; i++){
			fact =fact*i;
		}	

		System.out.printf("The factorial of 5 is : %d",fact);
	}	
}	
